lib/addons/cluster-autoscaler/index.ts

```diff
@@ -42,8 +42,9 @@ const defaultProps: ClusterAutoScalerAddOnProps = {
  * Version of the autoscaler, controls the image tag
  */
 const versionMap = new Map([
-    [KubernetesVersion.of("1.25"), "9.25.0"],
-    [KubernetesVersion.V1_25, "9.25.0"],
+    [KubernetesVersion.of("1.26"), "9.29.0"],
```

Comment:
Lines 45 and 46 look duplicate.

Reply:
i fully agree with you they look dublicate , but i tried to keep them as close as posssible before my commit. Check the lines in how they look into 1.8.1 with version for V1.25

Reply:
that is fine, as long as the map entry exists for the target version of EKS, it is the correct approach. Version may become different in the future.